src/high-level/matrix.lisp

```diff
@@ -522,3 +522,9 @@ NOTE: If MATRIX is not square, this will compute the reduced LQ factorization.")
         (magicl:@ vects
                   new-log-diag
                   (magicl:inv vects))))))
+
+(defmethod map!-lisp (function (tensor matrix))
+  ;; XXX: If we ever have a "stride" or the like, this could be
```

Comment:
What is "or the like" here? And can we error if the stride isn't 1?

Reply:
I wouldn't error, I would implement it properly